lte/gateway/c/oai/include/amf_app_messages_def.h

```diff
@@ -0,0 +1,50 @@
+/**
+ *  * Copyright 2020 The Magma Authors.
+ *  *
+ *  * This source code is licensed under the BSD-style license found in the
+ *  * LICENSE file in the root directory of this source tree.
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ * */
```

Comment:
Please use single asterisk at the beginning of multi-line comment

Reply:
okay